src/gui/speciestab_sites.cpp

```diff
@@ -157,24 +157,24 @@ void SpeciesTab::updateSitesTab()
     }
 
     // Set origin atom indices
-    Array<int> originAtoms = site->originAtomIndices();
-    QString originText = originAtoms.nItems() == 0 ? QString() : QString::number(originAtoms.at(0) + 1);
-    for (auto n = 1; n < originAtoms.nItems(); ++n)
+    std::vector<int> originAtoms = site->originAtomIndices();
+    QString originText = originAtoms.empty() ? QString() : QString::number(originAtoms.at(0) + 1);
+    for (auto n = 1; n < originAtoms.size(); ++n)
         originText += QString(" %1").arg(originAtoms.at(n) + 1);
     ui_.SiteOriginAtomsEdit->setText(originText);
     ui_.SiteOriginMassWeightedCheck->setCheckState(site->originMassWeighted() ? Qt::Checked : Qt::Unchecked);
 
     // Set x axis atom indices
-    Array<int> xAxisAtoms = site->xAxisAtomIndices();
-    QString xAxisText = xAxisAtoms.nItems() == 0 ? QString() : QString::number(xAxisAtoms.at(0) + 1);
-    for (auto n = 1; n < xAxisAtoms.nItems(); ++n)
+    std::vector<int> xAxisAtoms = site->xAxisAtomIndices();
+    QString xAxisText = xAxisAtoms.empty() ? QString() : QString::number(xAxisAtoms.at(0) + 1);
+    for (auto n = 1; n < xAxisAtoms.size(); ++n)
         xAxisText += QString(" %1").arg(xAxisAtoms.at(n) + 1);
```

Comment:
```suggestion
    for (auto idx = std::next(xAxiesAtoms.begin()); idx < xAxisAtoms.end(); ++idx)
        xAxisText += QString(" %1").arg(*idx + 1);
```

Just handling the loop through iterators instead of indices.  In case we don't go the `accumulate` route.